shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/resources/ManagedProcessor.java

```diff
@@ -0,0 +1,189 @@
+package com.redhat.service.smartevents.shard.operator.v2.resources;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.service.smartevents.infra.v2.api.models.dto.ProcessorDTO;
+import com.redhat.service.smartevents.shard.operator.core.utils.LabelsBuilder;
+import com.redhat.service.smartevents.shard.operator.core.utils.StringUtils;
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.client.utils.KubernetesResourceUtil;
+import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.ShortNames;
+import io.fabric8.kubernetes.model.annotation.Version;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.util.Objects.requireNonNull;
+
+@Group("com.redhat.service.smartevents")
+@Version("v2alpha1")
+@ShortNames("mproc")
+public class ManagedProcessor extends CustomResource<ManagedProcessorSpec, ManagedProcessorStatus> implements Namespaced {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ManagedProcessor.class);
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    public static final String COMPONENT_NAME = "executor";
+    public static final String OB_RESOURCE_NAME_PREFIX = "ob-";
```

Comment:
Wonder if we should update the prefix now to be sms (smartevents) or something similar? Openbridge is not really used any more.